README.md

```diff
@@ -28,7 +28,7 @@ doesn't comply with the license). Test files are considered source code.
 
 ### Apache header
 
-    Copyright 2021 Expedia, Inc.
+    Copyright [yyyy] Expedia, Inc.  <-- Remember to replace with the current year here
```

Comment:
Maybe we could automate this somehow, but I have no idea atm :).

Reply:
It could be automated with a text template and the variable injected by the build